gobblin-restli/gobblin-throttling-service/gobblin-throttling-service-server/src/main/java/gobblin/restli/throttling/LimiterServerResource.java

```diff
@@ -41,7 +43,9 @@
 @RestLiCollection(name = "permits", namespace = "gobblin.restli.throttling")
 public class LimiterServerResource extends ComplexKeyResourceTemplate<PermitRequest, EmptyRecord, PermitAllocation> {
 
-  @Inject
+  public static final String BROKER_INJECT_NAME = "broker";
+
+  @Inject @javax.inject.Inject @javax.inject.Named(BROKER_INJECT_NAME)
```

Comment:
In FlowConfigResource we use the following:
@Inject
@Named(...)

Are the javax ones equivalent to the google ones?